superset/databases/utils.py

```diff
@@ -104,7 +104,7 @@ def get_table_metadata(
     }
 
 
-def make_url_safe(raw_url: Union[str, URL]) -> URL:
+def make_url_safe(raw_url: Union[str, URL]) -> Any:
```

Comment:
This shouldn't be necessary?
```suggestion
def make_url_safe(raw_url: Union[str, URL]) -> URL:
```

Reply:
Good catch, thanks!